package.json

```diff
@@ -33,9 +33,11 @@
     "gatsby-background-image": "^0.10.2",
     "gatsby-cli": "^2.12.45",
     "gatsby-image": "^2.4.7",
+    "gatsby-plugin-google-analytics": "^2.3.14",
     "gatsby-plugin-google-tagmanager": "^2.3.5",
     "gatsby-plugin-hotjar": "^1.0.1",
     "gatsby-plugin-intl": "^0.3.3",
+    "gatsby-plugin-react-helmet": "^3.3.12",
```

Comment:
We have react helmet already. Which one is better? Could we delete one of them?

Reply:
I removed the gatsby-plugin since I wasn't using it actually anywhere :hand_over_mouth: